packages/app/src/__tests__/CorePack.test.tsx

```diff
@@ -19,7 +19,7 @@ test('visualise raw dataset', async () => {
   expect(tabs[0]).toHaveTextContent(Vis.Raw);
   expect(tabs[0]).toHaveAttribute('aria-selected', 'true');
 
-  expect(await screen.findByText(/"int": 42/)).toBeVisible();
+  await expect(screen.findByText(/"int": 42/)).resolves.toBeVisible();
```

Comment:
Galex enables a new rule called [jest/prefer-expect-resolves](https://github.com/jest-community/eslint-plugin-jest/blob/v25.3.0/docs/rules/prefer-expect-resolves.md), which brought this syntax to my attention.

> While the second style (ndlr. `expect(await...)`) is arguably less dependent on jest, if the promise rejects it will be treated as a general error, resulting in less predictable behaviour and output from jest.

In other words, Testing Library's errors should now be better displayed in Jest's output.